### PR TITLE
ArgoCD client set up

### PR DESCRIPTION
**Pull request by anil-intelops**
image
image
image
image

### CODE REVIEW

File: capten/common-pkg/plugins/argocd/client.go [hunk 1]
```diff
@@ -25,11 +26,31 @@ type ArgoCDCLient struct {
 }
 
 func NewClient(logger logging.Logger) (*ArgoCDCLient, error) {
-	cfg, err := fetchConfiguration(logger)
+	// cfg, err := fetchConfiguration(logger)
+	// if err != nil {
```
Comment (vramk23) [inline capten/common-pkg/plugins/argocd/client.go:30]: remove commented code

**Review by vramk23 — CHANGES_REQUESTED** [review]

File: capten/common-pkg/plugins/argocd/client.go (hunk 1 above)
Comment (vramk23) [reply capten/common-pkg/plugins/argocd/client.go:30]: lastupdatetime is empry. check this

File: capten/agent/pkg/capten-store/argocd_projects.go [hunk 2]
```diff
@@ -108,13 +108,15 @@ func (a *Store) updateArgoCDProjects() ([]*model.ArgoCDProject, error) {
...
+		if ap, ok := argoCDProjects[gitProject.Id]; !ok {
 			project.Status = string(model.ArgoCDProjectAvailable)
+			project.LastUpdateTime = time.Now().Format(time.RFC3339)
 			if err := a.UpsertArgoCDProject(project); err != nil {
 				return nil, err
 			}
 		} else {
 			project.Status = argoCDProjects[gitProject.Id].Status
+			project.LastUpdateTime = ap.LastUpdateTime
 		}
```
Comment (vramk23) [inline capten/agent/pkg/capten-store/argocd_projects.go:120]: add time is store method. check other table handling

Comment (anil-intelops) [reply capten/agent/pkg/capten-store/argocd_projects.go:120]: While inserting the record this time stamp is being added.

Same is being fetched and and returned with this api.

**Pull request merged**